packages/xo-web/src/xo-app/index.js

```diff
@@ -114,6 +116,17 @@ const BODY_STYLE = {
     signedUp: !!state.user,
   }
 })
+@provideState({
+  initialState: () => ({ checkXoaCount: 0 }),
+  effects: {
+    refreshXoaStatus() {
+      this.state.checkXoaCount += 1
+    },
+  },
+  computed: {
+    xoaStatus:  ({ checkXoaCount }) => checkXoa(),
```

Comment:
Don't you have an ESLint warning about `checkXoaCount` not being used?

Also, I think with this code, we risk that an aggressive minification tool will remove this destructuration, IMHO, we should explicitly use it like this:

```js
noop(checkXoaCount)
```

With a small comment explaining why we need to do this.

Reply:
No warning about unused `checkXoaCount` on my side.